minecraft_servers/domei/manifest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "server_name": "domei",
+  "nice_name": "DOMEI",
+  "direct_ip": "domei.eu",
+  "server_wildcards": [
+    "%.domei.eu"
+  ],
+  "supported_languages": [
+     "fr"
+  ],
+  "social": {
+    "web": "https://domei.eu/",
```

Comment:
1. Website is down (Error 521)
2. Remove "/" at the end

Reply:
For us, the website hasn't any problems. Tell us if it still appear offline on your side! :)